### PR TITLE
add ryan.json

### PR DESCRIPTION
**Pull request by MrRyanPerson**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [✅ ] The file is in the `domains` folder and is in the JSON format.
- [ ✅] The file's name is all lowercased and alphanumeric. 
- [✅] You have completed your website. 
- [✅] The website is reachable.  
- [✅] The CNAME record doesn't contain `https://` or `/`.  
- [✅] There is sufficient information at the `owner` field.  

## Website Link/Preview

ec2-3-12-165-61.us-east-2.compute.amazonaws.com

### CODE REVIEW

**Comment by epic-person-on** [comment]
note I will get ssl cert later

File: domains/ryan
Comment (DEV-DIBSTER) [inline domains/ryan:1]: File needs to end in .json.

File: domains/ryan [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "description": "Dev Portfolio",
+  "owner": {
+    "username": "epic-person-on",
+    "discord": "epic.person"
```
Comment (DEV-DIBSTER) [inline domains/ryan:5]: You're missing the email field.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]
Please read the docs: https://docs.is-a.dev/

**Comment by epic-person-on** [comment]
Fixed

**Comment by epic-person-on** [comment]
I added my email

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**